app/updates/BreakingNewsUpdate.scala

```diff
@@ -24,6 +24,85 @@ import scala.util.{Failure, Success, Try}
 
 class InvalidNotificationContentType(msg: String) extends Throwable(msg) {}
 
+object BreakingNewsUpdate {
+  val CovidGlobalTopicName = "global-covid-19"
+  val CovidBreakingNewsTopics = List(
+    BreakingNewsCovid19Uk,
+    BreakingNewsCovid19Us,
+    BreakingNewsCovid19Au,
+    BreakingNewsCovid19International
+  )
+
+  def createPayload(trail: ClientHydratedTrail, email: String): BreakingNewsPayload = {
+    val title = trail.topic match {
+      case Some("uk-general-election") => Some("General election 2019")
+      case Some(topic) if (CovidBreakingNewsTopics.map(_.name) :+ CovidGlobalTopicName).contains(topic) => Some("Coronavirus")
```

Comment:
`++` to `:+` is the only real change here – everything else is the move from class to object.